restore/data.go

```diff
@@ -203,6 +203,7 @@ func restoreDataFromTimestamp(fpInfo filepath.FilePathInfo, dataEntries []toc.Co
 		return 0
 	}
 
+	initialPipes := connectionPool.NumConns
```

Comment:
I doubt that `initialPipes` is a good name, as pipes are created only for single data file of cluster resize cases.
While we use this variable in all cases. Maybe better to leave it like `maxConnections` or smth similar?

Reply:
renamed